Bug when deploying to heroku: /breads

Logs
```
2021-12-23T04:18:29.932439+00:00 app[web.1]: Error: Cannot find module './layouts/Default'
2021-12-23T04:18:29.932452+00:00 app[web.1]: Require stack:
2021-12-23T04:18:29.932453+00:00 app[web.1]: - /app/views/index.jsx
2021-12-23T04:18:29.932453+00:00 app[web.1]: - /app/node_modules/express-react-views/index.js
2021-12-23T04:18:29.932454+00:00 app[web.1]: - /app/server.js
2021-12-23T04:18:29.932455+00:00 app[web.1]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
2021-12-23T04:18:29.932455+00:00 app[web.1]:     at Function.Module._load (internal/modules/cjs/loader.js:746:27)
2021-12-23T04:18:29.932455+00:00 app[web.1]:     at Module.require (internal/modules/cjs/loader.js:974:19)
2021-12-23T04:18:29.932456+00:00 app[web.1]:     at require (internal/modules/cjs/helpers.js:93:18)
2021-12-23T04:18:29.932456+00:00 app[web.1]:     at Object.<anonymous> (/app/views/index.jsx:2:23)
2021-12-23T04:18:29.932456+00:00 app[web.1]:     at Module._compile (internal/modules/cjs/loader.js:1085:14)
2021-12-23T04:18:29.932457+00:00 app[web.1]:     at Module._compile (/app/node_modules/pirates/lib/index.js:136:24)
2021-12-23T04:18:29.932457+00:00 app[web.1]:     at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
2021-12-23T04:18:29.932458+00:00 app[web.1]:     at Object.newLoader [as .jsx] (/app/node_modules/pirates/lib/index.js:141:7)
2021-12-23T04:18:29.932458+00:00 app[web.1]:     at Module.load (internal/modules/cjs/loader.js:950:32)
```

--- a/views/index.jsx
+++ b/views/index.jsx
@@ -1,5 +1,5 @@
 const React = require('react')
-const DefaultLayout = require('./layouts/Default')
+const DefaultLayout = require('./layouts/default')
 
 // Default is our html boilerplate wrapper imported from the layouts directory
 // Everything between <Default> and </Default> in the function below shows up as "html.children" in that component

